go.mod

```diff
@@ -91,7 +91,7 @@ require (
 	github.com/xlab/treeprint v1.1.0 // indirect
 	go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5 // indirect
 	golang.org/x/mod v0.10.0 // indirect
-	golang.org/x/net v0.10.0 // indirect
+	golang.org/x/net v0.33.0 // indirect; Fixes CVE-2024-45338
```

Comment:
There is a replace statement on line 120 that is making this change a no-op

Reply:
fixed